docs-src/cloud/temporal-platform-limits-sheet.md

```diff
@@ -0,0 +1,83 @@
+---
+slug: temporal-platform-limits-sheet
+title: What are the Temporal Platform limits?
+sidebar_label: Upgrade policy
+description: Basic upgrade policy for customers of Temporal Cloud
+tags:
+  - error
+  - warn
+  - limits
+---
+
+This section describes many of the default settings and limits that apply to associated parameters in Temporal Cloud.
```

Comment:
"... apply to many aspect of operating with Temporal Cloud" 

I don't think "associated parameters" is an ideal choice here.